src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1195,6 +1193,7 @@ private IAuthenticator GetAuthenticator(CommandOptions commandOptions)
 
         private Task HandleEncryptionKeysNotificationEvent(EncryptionKeysChangeParams @params, EventContext context)
         {
+            Logger.Verbose("EncryptionKeysNotificationEvent");

```

Comment:
We already log the incoming messages 

https://github.com/microsoft/sqltoolsservice/blob/16c7e323567023b08200ab35b2bfb2a5042385e0/src/Microsoft.SqlTools.Hosting/Hosting/Protocol/MessageDispatcher.cs#L211

Do we really need another log that's basically doing the same thing?

Reply:
The timing of logs can differ based on when the handler task begins (which is based on semaphore) so I see no harm in having both logs. It can help us investigate issues if parallel message processing blocks tasks for longer periods of time.

Reply:
What about adding the code to the message dispatcher instead? Saves having to add this line to every single request/event